editor/project_manager.cpp

```diff
@@ -2102,6 +2180,13 @@ void ProjectManager::_open_selected_projects_ask() {
 		dialog_error->popup_centered();
 		return;
 	}
+	// Check if the project is using features not supported by this build of Godot.
+	if (unsupported_features.length() > 1) {
```

Comment:
Shouldn't this be `unsupported_features.length() >= 1`?

Reply:
Yes, I must have forgotten to test 1-character features. Fixed.